mapiproxy/servers/default/asyncemsmdb/dcesrv_asyncemsmdb.h

```diff
@@ -4,6 +4,7 @@
    OpenChange Project
 
    Copyright (C) Julien Kerihuel 2015
+   Copyright (C) Carlos Pérez-Aradros Herce 2015
```

Comment:
I am uncomfortable with all these Author changes.

It is not your fault but I think OpenChange clearly needs to state what is the difference between an author and a contributor and how it reflects on the source code.

An author is a contributor to the file but a contributor is not necessary an author of the file.

From my perspective, an author should only be added to a file if the contribution is significant and contacting the author to be required or useful to understand the in-depth logic. Authoring is all about usefulness and authority for people looking at the code and who want to find answers.

 This change for example or the mpm_cache.c do not fit into this category imho. However I would definitely do a git blame and contact the contributor if I have questions about few lines of code.